src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -654,6 +654,41 @@ const schema = {
         },
       },
     },
+    specialIssues: {
```

Comment:
Hard to tell in this diff, but this is adding `specialIssues` as a definition because it'll be used three times.